### PR TITLE
Docs: hascal bash installer

### PR DESCRIPTION
**Pull request by mehanalavimajd**

### CODE REVIEW

File: docs/src/lang/0_install.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ Requirments :
 - `libcurl`,`libssl`,`libcrypt`
 
 *Nix :
+You can use the `hascal.sh` script to automate the process of installing and adding hascal to path:
```
Comment (mmdbalkhi) [inline docs/src/lang/0_install.md:8]: What is `hascal.sh`?

**Review by mmdbalkhi — CHANGES_REQUESTED** [review]

File: docs/src/lang/0_install.md (hunk 1 above)
Comment (mehanalavimajd) [reply docs/src/lang/0_install.md:8]: Checkout #62

**Review by mmdbalkhi — APPROVED** [review]

**Comment by mmdbalkhi** [comment]
@mehanalavimajd, thank you for contributing.

**Comment by mehanalavimajd** [comment]
@mmdbalkhi can you review #67 please?

**Pull request merged**